datasets/us/ar/med_exclusions/crawler.py

```diff
@@ -38,49 +36,32 @@ def crawl_item(row: Dict[str, str], context: Context):
         context.emit(person, target=True)
         context.emit(sanction)
 
-    if raw_facility_name := row.pop("Facility Name"):
+    if facility_name := row.pop("Facility Name"):
+        entity = context.make("LegalEntity")  # Sometimes the person's name.
+        entity.id = context.make_id(facility_name, zip_code)
+        entity.add("name", facility_name)
+        entity.add("country", "us")
+        entity.add("topics", "debarment")
+        entity.add("address", address)
         # The d/b/a is a person's name and then the company name
-        names = REGEX_DBA.split(raw_facility_name)
-        if len(names) == 2:
-            dba_person_name, facility_name = raw_facility_name[0], raw_facility_name[1]
-        else:
-            facility_name = raw_facility_name
-            dba_person_name = None
-            if len(names) != 1:
-                context.log.warning("More names than expected", raw_facility_name)
-
-        company = context.make("LegalEntity")  # Sometimes the person's name.
-        company.id = context.make_id(facility_name, zip_code)
-        company.add("name", facility_name)
-        company.add("country", "us")
-        company.add("topics", "debarment")
-        company.add("address", address)
-
-        if dba_person_name:
-            dba_person = context.make("Person")
-            dba_person.id = context.make_id(dba_person_name, zip_code)
-            dba_person.add("name", dba_person_name)
-            dba_person.add("country", "us")
-            dba_person.add("topics", "debarment")
-            link = context.make("UnknownLink")
-            link.id = context.make_id(company.id, dba_person.id)
-            link.add("object", company)
-            link.add("subject", dba_person)
-            link.add("role", "d/b/a")
-            context.emit(dba_person)
-            context.emit(link)
-
-        sanction = h.make_sanction(context, company)
+        if "d/b/a" in facility_name:
+            dba = context.lookup_value("names", facility_name)
```

Comment:
So `context.lookup_value` will identify a single value from the lookup. To get multiple values, use: 
```
result = context.lookup("names", facility_name)
if result is not None:
   result.values
```